misk/src/main/kotlin/misk/web/BoundAction.kt

```diff
@@ -110,6 +111,7 @@ internal class BoundAction<A : WebAction>(
     val initialSeedData = mapOf<Key<*>, Any?>(
       keyOf<HttpServletRequest>() to request,
       keyOf<HttpCall>() to httpCall,
+      keyOf<RequestContext>() to httpCall,
```

Comment:
Puts the `httpCall` instance into action scope with the `RequestContext` key in addition to the existing `HttpCall` key. This means cash libs can get a RequestContext instance out of action scope, instead of HttpCall, when they only need the fields provided by RequestContext.